todo_app/templates/index.html

```diff
@@ -9,10 +9,46 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <table class="table">
+        <h2>Items</h2>
+        <thead>
+          <tr>
+            <th scope="col">Task</th>
+            <th scope="col">Status</th>
+            <th scope="col"></th>
+            <th scope="col"></th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for item in items %}
+          <td>{{ item.title }}</td>
+          <td>{{ item.status }}</td>
+          <td>
+            <form action="/update_task?id={{ item.id }}&title={{ item.title }}&status={{ 'Completed' if item.status == 'Not Started' else 'Not Started' }}" method="post">
```

Comment:
Quite minor, but there's a fair amount of complexity being baked into this "action" field here - I think nicer to use hidden inputs and read from those rather than the parameters:
```suggestion
            <form action="/update_task?id={{ item.id }}&title={{ item.title }}&status={{ 'Completed' if item.status == 'Not Started' else 'Not Started' }}" method="post">
            <input name="id" value="{{ item.id }" hidden />
            <input name="title" value="{{ item.title}" hidden />
            <input name="status" value="{{ 'Completed' if item.status == 'Not Started' else 'Not Started' }}" hidden />
```

And then we would access with `request.form.get('<field_name>')` rather than `request.values.get('<field_name>')`